Monika After Story/game/definitions.rpy

```diff
@@ -360,6 +360,30 @@ python early:
         def __str__(self):
             return self.msg
 
+
+    # event data class - use for data grouping - no behaviors allowed
+    class EventDBData(object):
```

Comment:
Shouldn't we be calling this `MASEventDBData`?

Seeing the docstring for it, I *assume* this may see future use somewhere, in which case might be best to use the normal name guides for globals

Reply:
I kept it `EventDBData` since it is related to `Event`, so `MASEventDBData` might be confusing. For special core stuff we'll probably be fine with not using a MAS prefix.